### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Debugging_Screen reader:Narrator is reading the 'star on github' link as github star button 

### ISSUE THREAD

**Issue by Nikhitha-13** [microsoft/vscode-docs#2660]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment:**
URL: code.visualstudio.com/docs/editor/debugging
Browser: Microsoft Edge 44.17763.1.0​
Google Chrome Version 73.0.3683.103 (Official Build) (64-bit)

**NOTE:**
The issue is observed through out the application in the following URLs:
1. code.visualstudio.com/docs.
2. code.visualstudio.com/download.
3. code.visualstudio.com/docs/supporting/faq.
4. code.visualstudio.com/docs/setup/linux.
5. code.visualstudio.com/docs/getstarted/settings.
6. code.visualstudio.com/docs/python/python-tutorial.
7. code.visualstudio.com/updates/v1_30.
8. code.visualstudio.com/docs/editor/whyvscode.
9. code.visualstudio.com/docs/getstarted/tips-and-tricks.
10. code.visualstudio.com/updates/v1_29.
11. code.visualstudio.com/docs/editor/integrated-terminal.
12. code.visualstudio.com/docs/editor/debugging.
13. code.visualstudio.com/docs/editor/versioncontrol.

**Reprosteps:**
1. Hit the URL "code.visualstudio.com/docs/editor/debugging".
2. Tab till the star on github link which is present in footer section.
3. Start screen reader(Narrator).
4. Check how screen reader is reading the star on github link when focus lands on the link.

**Actual Result:**
When focus lands on star on github link, Narrator is reading as github star button star on github link.

**Expected Result:**
Screen reader should not read the star on github link as github star button. It should not announce the link as button.

**User Impact:**
Screen reader users will get confused about the role of the control if screen reader announces the role incorrectly.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by Nikhitha-13** [comment]
[Bug_#2660_Narrator reading as github star button star on github link.pptx](https://github.com/microsoft/vscode-docs/files/3162084/Bug_.2660_Narrator.reading.as.github.star.button.star.on.github.link.pptx)

**Comment by Nikhitha-13** [comment]
[Bug_#2660_narrator reading as github star button.pptx](https://github.com/microsoft/vscode-docs/files/3175944/Bug_.2660_narrator.reading.as.github.star.button.pptx)

**Comment by Nikhitha-13** [comment]
[Bug_#2660_Narrator reading as github star button star on github link.pptx](https://github.com/microsoft/vscode-docs/files/3175954/Bug_.2660_Narrator.reading.as.github.star.button.star.on.github.link.pptx)

**Comment by Nikhitha-13** [comment]
[Bug_#2660_Narrator reading as github star button star on github link bug.pptx](https://github.com/microsoft/vscode-docs/files/3175972/Bug_.2660_Narrator.reading.as.github.star.button.star.on.github.link.bug.pptx)

**Comment by gregvanl** [comment]
@Nikhitha-13 Fixed by removing unneeded title

**Comment by Nikhitha-13** [comment]
Yes The fix is available and the issue got fixed

**Comment by fsteffi** [comment]
#closed;

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
